install/update/class.taoQtiUpdate.php

```diff
@@ -73,9 +74,9 @@ protected function convertQtiItem(core_kernel_classes_Resource $item)
 
     protected function convertQtiFromV2p0ToV2p1($xml)
     {
-        
+
         $returnValue = '';
```

Comment:
In scope of current task, I don't want to fix things not related to PSR-12 🙂 
This will require additional time to find all of them.

But thanks for this 👍🏻

Reply:
There is a whiteline between the opening bracket for the method and the method body itself, and I think that line should be removed.

PSR-12 does not explicitly say that "the function body must not start with a whiteline", but IMO that is inferred from the examples in the specification. Also, PSR-2, which PSR-12 extends from, shows warnings for "Blank line found at start of control structure" ([ref](https://youtrack.jetbrains.com/issue/WI-41602)), and [section 5 of PSR-12](https://www.php-fig.org/psr/psr-12/#5-control-structures) states that "The body MUST be on the next line after the opening brace".

BTW I don't consider this a blocking issue for this PR to pass :slightly_smiling_face:

Reply:
@hectoras we are out of budget/time for the PSR-12, so if the build passes and the code is not broken, we can approve the PRs :)

Reply:
Yeah, I understand, I've already approved it.